src/main/java/core/strategy/DataOperationStrategyImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.strategy;
+
+import core.service.OperationType;
+import core.transactions.BalanceOperationHandler;
+import core.transactions.OperationHandler;
+import core.transactions.PurchaseOperationHandler;
+import core.transactions.ReturnOperationHandler;
+import core.transactions.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.Map;
+
+public class DataOperationStrategyImpl implements DataOperationStrategy {
+    @Override
+    public Map<OperationType, OperationHandler> strategy() {
+        Map<OperationType, OperationHandler> operationHandlers = new HashMap<>();
+        operationHandlers.put(OperationType.B, new BalanceOperationHandler());
+        operationHandlers.put(OperationType.S, new SupplyOperationHandler());
+        operationHandlers.put(OperationType.R, new ReturnOperationHandler());
+        operationHandlers.put(OperationType.P, new PurchaseOperationHandler());
+        return operationHandlers;
+    }
```

Comment:
Why you ignore comment about your strategy second time? We won't review your work until you fix it
"move this hashMap to main method and give it to constructor of your strategy"

Reply:
I didn't ignore. I thought that it will be better..

Reply:
If you did not fix the code and did not react to the comment in any way - it is called ignored
So if you think your solution is better - please, provide some arguments

Reply:
Please, read the discussion in slack before review